specs.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+# Specifications
+
+- [Design documents](https://github.com/WebAssembly/design): documents describing the design, goals and high-level overview of WebAssembly.
+- [Core specification](https://webassembly.github.io/spec/core/): defines the semantics of WebAssembly modules independent from a concrete embedding. The WebAssembly core is specified in a single document.
+- Embedding interfaces:
+  - [JavaScript API](https://webassembly.github.io/spec/js-api/index.html): defines JavaScript classes and objects for accessing WebAssembly from within JavaScript, including methods for validation, compilation, instantiation, and classes for representing and manipulating imports and exports as JavaScript objects.
```

Comment:
Some of the descriptions here have contents that could become out-of-date; I guess that's OK since we can always update them, but I wonder if it makes more sense to use a description here that doesn't really need to change.

Reply:
FWIW these descriptions are pretty much copy-pasted from https://webassembly.github.io/spec/ which we were pointing to before.

If they're out of date, it's probably worth updating them at the source as well, but I'm happy to accept any fixes here :)

Reply:
Ah, fair enough :) They're accurate enough for now, lets keep them as is.